src/loader-components/PropsTable.module.scss

```diff
@@ -17,12 +17,12 @@
     word-wrap: break-word;
   }
 
-  tr {
+  .tr {
     background-color: var(--terra-dev-site-props-table-tr-background-color, #fff);
     border-top: 1px solid #c6cbd1;
   }
 
-  tr:nth-child(2n) {
+  .tr:nth-child(2n) {
```

Comment:
A little confusing the class name is the tag name, but I double checked that this is the correct class being bound

Reply:
Thats probably my fault 😁

Reply:
will this break table's screenshots once this is released?